.github/dependabot.yml

```diff
@@ -16,8 +16,14 @@ updates:
       - dependency-name: "django"
         versions:
           - "> 4.2"  # This ignores versions above 4.2, allowing only 4.2.x LTS upgrades
+    groups:
+      all-python-dependencies:
+        pattern: "*"
```

Comment:
In the[ referenced resource page](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/customizing-dependency-updates#grouping-dependabot-updates-into-one-pull-request), only `patterns` is used, not pattern. I am not sure whether we need to change to:

`patterns:
-*
`

Reply:
Yes, this is changed thanks!